### PR TITLE
Add eslint 6 support

### PR DESCRIPTION
**Pull request by HDuck**
Fixed tests to check eslint ver.6. Added eslint-6 support.

### CODE REVIEW

**Comment by HDuck** [comment]
Check failes on Node.js 6

**Comment by graue** [comment]
This patch looks good. ESLint 6 [no longer supports Node.js 6](https://eslint.org/docs/user-guide/migrating-to-6.0.0#drop-node-6), so the CI build failing for Node 6 is expected. Probably need to update `.travis.yml` to install ESLint 5 when running on Node 6.

**Comment by platinumazure** [comment]
Hi everyone, I'm from ESLint.

Instead of doing a direct require to a formatter path, I would recommend `CLIEngine#getFormatter` (if you have a CLIEngine instance) or `CLIEngine.getFormatter` (static function you can use if you don't have an instance).

Requiring the path directly is not part of the public API and could change any time, even in patch releases. The two `CLIEngine` options are in the public API and won't change except in a major release (and then, only when absolutely needed and we can't deprecate).

Good luck! Please stop by our [Gitter chat](https://gitter.im/eslint/eslint) if you need help.

**Comment by lochstar** [comment]
@HDuck See https://github.com/webpack-contrib/eslint-loader/issues/238#issuecomment-505672107 regarding the recommended approach for the formatter options.

**Comment by HDuck** [comment]
I'll fix it all today. Thanks for assisting. If anyone knows how to change travis config to install eslint 5 on node 6 I'll appreciate additional help. ;)

**Comment by Sunshine168** [comment]
@HDuck I have a suggestion which drop node 6 in travis.yml file and add node engines require in package.json { "engines" : { "node" : "> 6" } }  ,then update the doc. Because for Eslint 6 , Node.js 6 is no longer supported it .

**Comment by vinayakkulkarni** [comment]
Fixes #271 

This PR would fix the `stylish` formatter issue

**Comment by evilebottnawi** [comment]
Ping me when the PR was ready for review

**Comment by HDuck** [comment]
@evilebottnawi ready.

File: package.json [hunk 1]
```diff
@@ -57,5 +57,8 @@
       "test/*.js"
     ],
     "verbose": true
+  },
+  "engines": {
+    "node": ">6"
```
Comment (evilebottnawi) [inline package.json]: Drop this from this PR

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   },
   "devDependencies": {
     "ava": "^0.17.0",
-    "eslint": "^5.1.0",
+    "eslint": "5.1.0 - 6",
```
Comment (evilebottnawi) [inline package.json]: use `6.0.1`, let's test all on latest stable release

File: package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": ">=1.6.0 <6.0.0",
+    "eslint": ">=1.6.0 <7.0.0",
```
Comment (evilebottnawi) [inline package.json]: Not sure this loader works fine with `eslint@1`/`eslint@2`/`eslint@3`/`eslint@4`, but it is out of scope this PR

Comment (HDuck) [reply package.json]: Checked on `eslint@^4`, `eslint@^5`, `eslint@^6`. Tests is passing.

File: package.json (hunk 2 above)
Comment (HDuck) [reply package.json]: Ok. Changed on `^6.0.1` version.

File: package.json (hunk 1 above)
Comment (HDuck) [reply package.json]: Done

**Comment by HDuck** [comment]
@evilebottnawi Tested manually `eslint@^4` and `eslint@^5` versions. All is okay.

**Comment by lochstar** [comment]
Should the README be updated to reflect the new usage?

**Pull request merged**